plugins/headlamp-plugin/bin/headlamp-plugin.js

```diff
@@ -397,6 +397,33 @@ async function createArchive(pluginDir, outputDir) {
   return 0;
 }
 
+/**
```

Comment:
vite has this functionality built-in https://vite.dev/config/shared-options.html#envprefix

you can add this to vite.config.mjs:
```
envPrefix: ['NODE_ENV', 'REACT_APP_', 'HEADLAMP_APP_'],
```

Reply:
Oh cool. Let me have a look...